### DISCUSSION TITLE
2022-12-14 Weekly Meeting 86 Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#324] [General]
Weekly Meeting 86 is happening on Wednesday, December 14, 2022 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 

- OMC updates next week (moved to 12/21/22) #309 
  - **Tracking two main items related to this:** 
    - Online element: Watch parties, streaming capabilities, virtual participation (like Q+A) 
    - Ticket prices: Particularly for the virtual element... can we make it easier to make the price free or almost free? Maybe a promo code, or the watch parties could allow people to attend without needing a ticket at all. Furthermore, for anyone who may be volunteering in-person at the conference (like our friends at Croquet have volunteered to do) we need to ensure they have a in-person ticket as well. Specifically, these things tend to result in open source builders being exploited for _free work_. This group does not intend to be working for free. 
- This is a time to build. What can we build?
  - Adding OMI to the Metaverse Street Jin demoed
  - M3 recently started a shop for wearables. This is a great example of things we could be doing and participating in. 
  - Let's build right now. 

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo (December 7!)
- `IPSME` and Me
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
## Physics Progress 

The progress is going well, and we will check in after intros! 

Last week #323 we had a demo of some of our draft protocols. Today we are checking in! 

Questions: 
- what are the next steps? 
  - put this in front of some of the MSF working groups too. Since they are at the beginning, and if they see something like this, that can help highlight and emphasize that this work is going on in the field and they don't have to reinvent the wheel. 
    - For example like how Microsoft made Microsoft physics and theirs is not as far along as ours. We could raise more awareness of our drafts and collaborate with other bodies to reduce noise and spinning wheels. 
- Who should be in charge of pushing these things upstream to other bodies? 
  - need management help sharing with other groups. 
  - we have a process for advancing protocols. 
    - gLTF group meets tomorrow at 2:30 PT. 
  - we haven't made it exactly clear what the process is for contributing that work. 
  - the gLTF group is seeking additional leadership with bandwidth, specifically to help ensure things like our process are in order, and that we are getting the word out about progress and implementation.

**Reply by mrmetaverse** [reply]
"What does implementers mean?"
"didn't someone last week say they are a Unity expert and can help implement?"

**Reply by mrmetaverse** [reply]
# CTA - HOW YOU CAN HELP

We are always looking for groups to test, experiment with, and implement our proposed extensions. (It's also part of our standard process for considering an extension or protocol as "ready"). 

We would love to work together with other experts, like an Unreal expert, and Unity expert to see what we could change or edit.

**Comment by mrmetaverse** [comment]
## OpenSim Avatar Conference was this past weekend
- about 280 attendees 
  - about 170 registered
  - the rest were able to come in freely from other servers due to the hypergrid. 
- Lot's of discussion about OMI, our working groups, and the MSF. 
- Links: 
  - Here's the link to the Open Metaverse panel @ OSCC (via Mal Burns' youtube): https://www.youtube.com/watch?v=X9WqJMf9cK8

**Comment by mrmetaverse** [comment]
## Website 
- Shadow would like to update the website with more images

**Comment by mrmetaverse** [comment]
# FIX THE BOTS :)  #325
- kill the old milestone, 
- kill any old tickets 
- start 1 new one. Finish it. 
- Profit :heavy_check_mark:

**Comment by mrmetaverse** [comment]
## Field trip planning with M3
- Metatraversal coming up, ask for details from @madjin 
- https://github.com/M3-org/research/issues/15
- Saturday Dec 17th 2pm EST - Planning session

**Comment by mrmetaverse** [comment]
## Next week: 

- Q&A and planning with L1 reps about the [OMC](https://openmetaverseconf.com)
- Define the obligations, expectations, and scope. What are we doing? Re-write it here: https://github.com/omigroup/omigroup/issues/309

**Comment by RedEagleP1** [comment]
I feel so bad that I am missing out on all the action now that our P1 meeting is now at the same time as OMI (due to DST).

**Reply by mrmetaverse** [reply]
We miss ya 🤟
